src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1732,4 +1733,22 @@ public boolean hasResponseToInstructorOrGeneral() {
         }
         return false;
     }
+
+    /**
+     * Returns list of responses with identities of giver/recipients NOT hidden which is used for
+     * anonymous result calculation.
+     * @param question question whose responses are required
+     * @return List of responses
+     */
+    public List<FeedbackResponseAttributes> getActualResponses(
+            FeedbackQuestionAttributes question) {
+        String questionId = question.getId();
```

Comment:
@xpdavid  I was wrong. This method makes use of `actualResponses` attribute of `FeedbackSessionResultsBundle` to get responses with identities of giver/recipients NOT hidden. 
```
    /**
     * Responses with identities of giver/recipients NOT hidden.
     * To be used for anonymous result calculation only, and identities hidden before showing to users.
     */
    public List<FeedbackResponseAttributes> actualResponses;
```

Should I add unit test for this method?

Reply:
Of course :)